Remotion/Web/Core/Res/Themes/NovaViso/HTML/WebButton.css

```
@@ -2,108 +2,55 @@
 
 button.webButton
 {
-  background-color: Transparent;
-  cursor: hand;
+  background-color: transparent;
+  border: var(--remotion-themed-border-width) solid var(--color-primary);
+  border-radius: var(--remotion-themed-border-radius);
+  padding: 0.4em 1.2em;
```

the paddings get updated with the css properties from the RM-7911 BocTextValue PR. to be reviewed next.
```suggestion
  padding: calc(3 * var(--remotion-themed-padding)) calc(3 * var(--remotion-themed-padding));
```